ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rhcs.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-rosa-sts-rhcs-full-f3
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: nightly
+      CLUSTER_PROFILE: rosa-sts-ad
+      RHCS_ENV: staging
+    test:
+    - ref: rhcs-e2e-tests
+    workflow: rhcs-aws-sts
+zz_generated_metadata:
+  branch: master
+  org: openshift
+  repo: openshift-tests-private
```

Comment:
@xueli181114 Will this job run also openshift-tests-private repository cases? or only terraform-provider-rhcs repository TCs?

Reply:
Only terraform-provider-rhcs  cases. I put it here just for future report portal usuage

Reply:
I think better to put it under https://github.com/openshift/release/tree/5d1f9c75140f63481edbb79bbb897c1a27c61d22/ci-operator/jobs/terraform-redhat , as it is not related to openshift-tests-private repo

Reply:
Let's put it here until I get aligned with OCP team to know how to use their report portal